README.md

```diff
@@ -82,14 +101,6 @@ An example of the what the contents of the IRIDA Next JSON file looks like for t
 
 Within the `files` section of this JSON file, all of the output paths are relative to the `outdir`. Therefore, `"path": "adjust/SAMPLE1_S_bracken_abundances.csv"` refers to a file located within `outdir/adjust/SAMPLE1_S_bracken_abundances.csv`.
 
-## Test profile
-
-To run with the test profile, please do:
-
-```bash
-nextflow run phac-nml/speciesabundance -profile docker,test -r dev -latest --outdir results
-```
-
 # Legal
 
 Copyright 2023 Government of Canada
```

Comment:
Copyright 2024
(I know, we need to update a lot of these!)

Reply:
Ah yes - great catch :smiley: [dd6bd99](https://github.com/phac-nml/speciesabundance/pull/11/commits/dd6bd99923b39487ec5bca0d8d976dfea9b2e88c)